File: Code/Prototype/Version_5/WebApplication1/Scripts/App/Model/PersonalData/PatientAlergyViewModel.js
function PatientAlergyViewModel(data) {
    var self = this;

    this.AlergyId = ko.observable(data.AlergyId);
    this.AlergyName = ko.observable(data.AlergyName);
    this.Description = ko.observable(data.Description);
    this.WhenDiagnosed = ko.observable(data.WhenDiagnosed);

};

File: Code/Prototype/Version_5/WebApplication1/ViewModels/PatientAlergyViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.ViewModels
{
    public class PatientAlergyViewModel
    {
        public int AlergyId { get; set; }
        public string AlergyName { get; set; }
        public string Description { get; set; }
        public DateTime WhenDiagnosed { get; set; }
    }
}